### PR TITLE
ar71xx: add GL-AR150 support

### PR DESCRIPTION
**Pull request by jannic**
I added support for GL-AR150 because a friend bought such a device and wanted to use it with Freifunk.

The device worked with branch v2016.1.x, and the same patch also applies to master. Unfortunately, he broke the router (via static discharge, not software-related), so I'm unable to test it with a current gluon version. Therefore, for now, I marked the device as 'BROKEN'.

### CODE REVIEW

File: patches/openwrt/0095-ar71xx-add-GL-AR150-support.patch [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+ CONFIG_ATH79_MACH_F9K1115V2=y
++CONFIG_ATH79_MACH_GL_AR150=y
+ CONFIG_ATH79_MACH_GL_INET=y
+ CONFIG_ATH79_MACH_GS_MINIBOX_V1=y
+ CONFIG_ATH79_MACH_GS_OOLITE=y
+diff --git a/target/linux/ar71xx/generic/profiles/gl-connect.mk b/target/linux/ar71xx/generic/profiles/gl-connect.mk
+deleted file mode 100644
+index e9377db..0000000
+--- a/target/linux/ar71xx/generic/profiles/gl-connect.mk
++++ /dev/null
```
Comment (NeoRaider) [inline patches/openwrt/0095-ar71xx-add-GL-AR150-support.patch]: Why is this file removed? We actually use the GLINET profile...

File: patches/openwrt/0095-ar71xx-add-GL-AR150-support.patch [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+index 5768286..718ee0b 100755
+--- a/target/linux/ar71xx/base-files/lib/ar71xx.sh
++++ b/target/linux/ar71xx/base-files/lib/ar71xx.sh
+@@ -458,6 +458,9 @@ ar71xx_board_detect() {
+ 		name="gl-inet"
+ 		gl_inet_board_detect
+ 		;;
++	*"GL AR150")
++        name="gl-ar150"
++        ;;
```
Comment (NeoRaider) [inline patches/openwrt/0095-ar71xx-add-GL-AR150-support.patch]: Indentation should use tabs instead of spaces, as the other entries.

Comment (jannic) [inline patches/openwrt/0095-ar71xx-add-GL-AR150-support.patch]: Taken verbatim from upstream. I could fix that, of course.

File: patches/openwrt/0095-ar71xx-add-GL-AR150-support.patch (hunk 1 above)
Comment (jannic) [inline patches/openwrt/0095-ar71xx-add-GL-AR150-support.patch]: Upstream moved the definition from gl-connect.mk to gli.mk. I'm not sure why my commit is missing that file.
Probably I make some mistake when doing 'make update-patches'.
Can't reconstruct it now, as it was done several  weeks ago. But I'll have a look.
(In fact, I wonder why that didn't lead to an error when compiling gluon?)

Comment (NeoRaider) [inline patches/openwrt/0095-ar71xx-add-GL-AR150-support.patch]: OpenWrt behaves a bit weird when profiles are missing - it will just make the non-existing profile behave like the "default" profile that contains all images. Build will take a lot longer, but it won't cause an error...

Comment (jannic) [inline patches/openwrt/0095-ar71xx-add-GL-AR150-support.patch]: Ok that's a good explanation - and the files are probably missing because I forgot to git-add them after applying the upstream patch. (just git apply ...; git commit -a)

Comment (jannic) [inline patches/openwrt/0095-ar71xx-add-GL-AR150-support.patch]: I regenerated the patch to contain the missing files. (See below.)
